.clang-format

```diff
@@ -18,6 +18,7 @@ SpacesBeforeTrailingComments: 2
 IndentAccessModifiers: false
 Cpp11BracedListStyle: false
 BreakBeforeBraces: Custom
+ColumnLimit: 0
```

Comment:
Why do we need this change? If it's just to prevent breaking long strings, maybe we can set `BreakStringLiterals: false`?

I think it also breaks the clang-format check in pre-commit: https://github.com/seladb/PcapPlusPlus/actions/runs/9989182798/job/27607221969?pr=1456

Reply:
I see, let me change to use `BreakStringLiterals: false`